app/streams/publishing_api/item_handler.rb

```diff
@@ -19,10 +19,14 @@ def process_links!
 
   def links_have_changed?
     return true if old_item.nil?
+    return false if new_item.nil?
+
+    old_links = old_item.expanded_links ? old_item.expanded_links : {}
```

Comment:
This will work, but I'm wondering whether we should set `{}` as the default for `expanded_links`. 

Would mean migrations to update the column, add that default for previous data that currently have `nil`, and adding an `attributes` default on the model (maybe). Maybe for a separate PR though. But I'm not really sure that's a good approach, so want to know what are your thoughts?

Reply:
Weirdly the default value is already set to `{}` which could suggest that our data is dodgy, however I think it is worth pushing this to fix the errors in sentry for now.

Reply:
Yep thanks for looking. Feel free to merge for now. As I said, we can look at that later.